src/utils/types.ts

```diff
@@ -24,3 +24,18 @@ export interface ContactType {
     date: string;
   };
 }
+
+export interface UserType {
+  email: string;
+  name: string;
+  pic: string;
+  username: string;
+}
+
+export enum UsersFieldsType {
```

Comment:
It's redundant to add words like Type or Interface when Typing. **UsersFields** is enough.
String enums should be capitalized by convention:

Email = 'email'
Name = 'name'

so you use them like UsersFields.Email